_data/docs.yml

```diff
@@ -7,16 +7,15 @@
       # - Link/Filename: Displayname
```

Comment:
Before we list the components we should point to the architectural overview. In other words please move the overview section right above the components section (the YAML for this looks like it can brake things so be aware of this).